apps/backend/src/app/api/v1/team-memberships/[team_id]/[user_id]/route.tsx

```diff
@@ -1,4 +1,5 @@
 import { teamMembershipsCrudHandlers } from "../../crud";
 
+// TODO: move this to /team-memberships
```

Comment:
What is the repos policy with todos? Do we get to them next time the code with a todo changes? Put an issue together when a todo is created?

Reply:
We have an internal task board to track all these, and we only publish part of the easy-to-understand tasks to the public GitHub issues (this one for example is not super interesting to most of the developers). TODOs in the code are just reminders when we work on the same code.